### DISCUSSION TITLE
为PCL2添加节日风格的特殊启动画面

### DISCUSSION THREAD

**Discussion by CiclesMiao** [Meloong-Git/PCL#3081] [讨论]
我觉得可以给PCL2启动的图标增添一些节日暖心细节，或是让用户自定义修改打开启动器时显示的PCL图标。
因为我喜欢樱花，所以魔改了PCL2的ICO图标使其变得更加好看，这是png版的:

![PCL2ICO](https://github.com/Hex-Dragon/PCL2/assets/103619120/1dbf5cb5-d6e8-4b36-b9a7-056daf4463fc)

作者可以考虑在每年3月15日到4月15日之间，樱花盛开的时候，使用我二创制作的启动图标，可以加入是否启用节日风PCL2启动图标的选项，顺便加入让用户自定义启动图标的选项吧...

**Comment by MoYuan-CN** [comment]
龙猫说过不会修改启动图标样式来着...
我找下帖

不过这二创挺好看 :)

**Reply by CiclesMiao** [reply]
只是建议一下，不行就算了

**Reply by MoYuan-CN** [reply]
找不到帖子了...
我记得图标是打包在 EXE 里的，好像不太好实现调本地文件来展示。~~万一本地图片文件炸了....~~

**Reply by kong-que-dnf123** [reply]
https://github.com/Hex-Dragon/PCL2/issues/791#issuecomment-1370479332
>被处理几率不大，无论从必要性还是从技术层面讲。（如果这样的话，难不成还可以通过 自定义 主题得到一个炫彩PCL2？233333）

**Reply by Deep-Dark-Forest** [reply]
> 找不到帖子了...
> 
> 我记得图标是打包在 EXE 里的，好像不太好实现调本地文件来展示。~~万一本地图片文件炸了....~~

好像没那么容易炸吧……除非有人改包

**Reply by MoYuan-CN** [reply]
> 除非有人改包

我说调本地图来的时候

**Reply by Deep-Dark-Forest** [reply]
> > 除非有人改包
> 
> 我说调本地图来的时候

把所有图标像蓝色默认图标一样放在exe里不就得了

**Comment by WForst-Breeze** [comment]
#791 已经提及不会支持启动器图标随主题的变化，可以推算出大概率同样不会制作您提及的内容。不过如果您觉得这个内容真的很酷很不错也可以交一个反馈，不过拒绝的几率比较大就是了😥
